python/nvtx/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import os
+
+from nvtx.nvtx import _annotate_nop, annotate, pop_range, push_range, mark
+
+if os.getenv("PYNVTX_DISABLE"):
+    annotate = _annotate_nop
```

Comment:
Do we want to keep this code? I don't think we saw any real change / benefit from it.

Reply:
Nope - deleted